### DISCUSSION TITLE
miss other polygons after export skin

### DISCUSSION THREAD

**Discussion by Delw1ze** [Parik27/DragonFF#269] [Q&A] [answered]
why i export skin(fam3.dff) from blender with plugin dragonff, and my skin dont have different polygons?(check cap in the screenshot)
![sa-mp-022](https://github.com/Parik27/DragonFF/assets/155895943/7b091e9d-a2d5-4360-9f82-cb58f3d01b26)

**Comment by YOEL-44** [comment, marked as answer]
I would first of all check if those polygons are flipped, San Andreas has backface culling, so you will not be able to see those.
